File: skills/astra-toolkit/clients/python/examples/collections-insert-many-lexical.py
import os
from astrapy import DataAPIClient

# Get an existing collection
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)
collection = database.get_collection("**COLLECTION_NAME**")

# Insert documents
result = collection.insert_many(
    [
        {
            "name": "Jane Doe",
            "$lexical": "An author who writes SciFi and fantasy novels.",
        },
        {
            "name": "Mary Day",
            "$lexical": "An active hiker, runner, and triathlete who loves the outdoors.",
        },
    ]
)

File: skills/astra-toolkit/clients/java/examples/collections-insert-many-lexical.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.collections.Collection;
import com.datastax.astra.client.collections.definition.documents.Document;

public class Example {

  public static void main(String[] args) {
    // Get an existing collection
    Collection<Document> collection =
        new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN"))
            .getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"))
            .getCollection("**COLLECTION_NAME**");

    Document document1 =
        new Document()
            .append("name", "Jane Doe")
            .append("$lexical", "An author who writes SciFi and fantasy novels.");
    Document document2 =
        new Document()
            .append("name", "Mary Day")
            .append("$lexical", "An active hiker, runner, and triathlete who loves the outdoors.");

    collection.insertMany(document1, document2);
  }
}
